authors/apps/articles/models.py

```diff
@@ -47,6 +49,26 @@ def save(self, *args, **kwargs):
                                 uuid.uuid4().hex[:6])
         super().save(*args, **kwargs)
 
+    @property
+    def get_time_to_read(self):
+        # Set the standard read time
+        words_per_min = settings.WORDS_PER_MIN
+        """
+        Cleaning the post content and lower casing all words: 
+        """
+        for char in '-.,\n':
```

Comment:
@pngg this is an expensive operation. Replace it with regex. Resource: https://stackoverflow.com/questions/875968/how-to-remove-symbols-from-a-string-with-python

Reply:
I have implemented the recommendation 👍